src/assets/styles/SensumTheme.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
this file is no longer needed =)

Reply:
Nice, didn't know whether you wanted it there or not.